mediagrains/comparison/_internal.py

```diff
@@ -256,7 +256,7 @@ def compare(self, a, b):
         if self.d.ratio() == 1.0:
             return (True, "Binary data {} are equal".format(self._identifier.format('<a/b>')), [])
         else:
-            first_op = [x for x in self.d.get_opcodes() if x[0] is not 'equal'][0]
+            first_op = [x for x in self.d.get_opcodes() if x[0] != 'equal'][0]
```

Comment:
So I think this matters because `x[0]` won't necessarily point to the same object as 'equal', but I don't understand why immutable timestamps changed it?

Reply:
It didn't, but I got a linting complaint on it with flake8 so I fixed it.